src/server/jobQueuesUI.js

```diff
@@ -20,7 +20,13 @@ const toureiroApp = toureiro({
 app.use(
   '/job-queues',
   (req, res, next) => {
-    if (!req.user || !userCan(req.user, 'monitorJobQueues')) {
+    const baseUrl = req.baseUrl
+    const encodedURI = encodeURIComponent(`${process.env.APP_BASE_URL}${baseUrl}`)
+
+    if (!req.user) {
+      res.redirect(`${process.env.IDM_BASE_URL}/sign-in?redirect=${encodedURI}`)
```

Comment:
This same URL is also composed in `src/common/routes/index.jsx`. Let's pull it into a util function used both here and there, perhaps in a module `src/common/util/auth.js`.

We also want the user to be redirected _back_ to the job queues path instead of to home/root as it's currently doing here.

You might wanna have something like:
```js
// in src/common/util/auth.js
export function loginURL(options = {}) {
  const redirect = options.redirect ? `?redirect=${encodeURIComponent(options.redirect)}` : ''
  return `${process.env.IDM_BASE_URL}/sign-in${redirect}`
}
```

And then do:
```js
// in src/server/jobQueuesUI.js
import {loginURL} from 'src/common/util/auth'
...
if (!req.user) {
  const redirectUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`
  res.redirect(loginUrl({redirect: redirectUrl}))
}
...
```

```js
// in src/common/routes/index.jsx
import {loginURL} from 'src/common/util/auth'
...
    if (typeof window !== 'undefined' && window.location) {
      window.location.href = loginURL({redirect: window.location.href})
    }
...
```